GVRf/Framework/framework/src/main/java/org/gearvrf/GVRRenderBundle.java

```diff
@@ -38,13 +40,16 @@
                 sampleCount = maxSampleCount;
             }
         }
-
         if (sampleCount <= 1) {
-            mPostEffectRenderTextureA = new GVRRenderTexture(mGVRContext, width, height);
-            mPostEffectRenderTextureB = new GVRRenderTexture(mGVRContext, width, height);
+            if(appSettings.isMultiviewSet())
+                mEyeCaptureRenderTexture  = new GVRRenderTexture(mGVRContext, width, height, 1);
```

Comment:
Suggest to create the capture render texture on demand. The capture is rarely used so we shouldn't be using up the memory all the time.